models/entry.go

```diff
@@ -95,3 +95,13 @@ func (e *Entry) IsValuesEmpty() bool {
 func (e *Entry) HasReason() bool {
 	return e.NegationReason != (CodedConcept{}) || e.Reason != (CodedConcept{})
 }
+
+func (e *Entry) WrapResultValue(val ResultValue) ResultValueWrap {
+	var rvWrap = ResultValueWrap{EntryWrap: e, Values: []ResultValue{val}}
+	return rvWrap
+}
+
+func (e *Entry) WrapResultValues(vals []ResultValue) ResultValueWrap {
+	var rvWrap = ResultValueWrap{EntryWrap: e, Values: vals}
+	return rvWrap
```

Comment:
this method and its sibling above can both be shortened like:
`return ResultValueWrap{EntryWrap: e, Values: vals}`